### DISCUSSION TITLE
Wlan Pi Pro radiation pattern

### DISCUSSION THREAD

**Discussion by bmisra01** [WLAN-Pi/feedback#37] [Hardware - WLAN Pi Pro] [answered]
Is there any documentation about radiation pattern? I see that RSSI of AP varies by almost 10-15 dB based on orientation (best is when the PI is standing UP v/s lying down)

**Comment by jolla** [comment, marked as answer]
Yes, that is correct, the antenna design in the WLAN Pi Pro was intended to be used standing up. The antenna datasheet with radiation patterns can be found here. 

https://www.te.com/commerce/DocumentDelivery/DDEController?Action=showdoc&DocId=Data+Sheet%7FDatasheet_PN-2344655-X%7FB2%7Fpdf%7FEnglish%7FENG_DS_Datasheet_PN-2344655-X_B2.pdf%7F2344655-7

P/N: 2344655-7

**Comment by bmisra01** [comment]
Thanks Jolla
